### ISSUE TITLE
Misaligned output when job output is big. 

### ISSUE THREAD

**Issue by fgutierrezz** [rundeck/rundeck#5172]
**Describe the bug**
The job output is misaligned when the output is too big 

**My Rundeck detail**

* Rundeck version: 3.1
* install type: Launcher
* OS Name/version: [e.g. ubuntu 16.04]
* DB Type/version: [choose: mysql, postgres, mssql, oracle, ... ]

**To Reproduce**
1. Create a job with big echo output

Ex: 

echo "asdfffddddddddddddddddddddddasfasdfsadfsadfsdadasfdfasdfsfdsafdsadfasfdsadfasdafsdfasdfasdafsadfadsfwqefwqefqwefwqefwefwefwefwefwefsdfsdfsdfsdfsdfsdfsdfdsdfsdf"

2. Open the Job output in Activity and check the Node > job output 

**Expected behavior**
It's expected the output to be aligned 

**Screenshots**
Attached
![BigOutput](https://user-images.githubusercontent.com/30966360/63374101-27464b00-c357-11e9-94a7-fd44638ebdee.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser Chrome
 - Version 76.0.3809.100

**Additional context**
Job attached . 
```
- defaultTab: nodes
  description: ''
  executionEnabled: true
  id: a2b10272-0585-44fd-aa16-684845c77448
  loglevel: INFO
  name: ansible_output_test
  nodeFilterEditable: false
  scheduleEnabled: true
  sequence:
    commands:
    - script: echo "asdfffddddddddddddddddddddddasfasdfsadfsadfsdadasfdfasdfsfdsafdsadfasfdsadfasdafsdfasdfasdafsadfadsfwqefwqefqwefwqefwefwefwefwefwefsdfsdfsdfsdfsdfsdfsdfdsdfsdf"
    - exec: echo "asdfffddddddddddddddddddddddasfasdfsadfsadfsdadasfdfasdfsfdsafdsadfasfdsadfasdafsdfasdfasdafsadfadsfwqefwqefqwefwqefwefwefwefwefwefsdfsdfsdfsdfsdfsdfsdfdsdfsdf"
      plugins:
        LogFilter:
        - config:
            appendToFile: 'false'
            fileDestination: /home/rundeck/instancias/310_ldap/logs_ansible.txt
          type: SaveToFile
    keepgoing: false
    strategy: node-first
  uuid: a2b10272-0585-44fd-aa16-684845c77448
```

**Comment by carlosrfranco** [comment]
Tested on 3.1.x:

![issue_5172_ok](https://user-images.githubusercontent.com/7209908/71586128-db22c880-2af7-11ea-8c33-06c9d65e1f36.png)

**Comment by carlosrfranco** [comment]
fixed on 3.1.4 and 3.2.x
